libnetwork/datastore/datastore.go

```diff
@@ -70,16 +70,17 @@ var (
 
 const defaultPrefix = "/var/lib/docker/network/files"
 
-// DefaultScope returns a default scope config for clients to use.
-func DefaultScope(dataDir string) ScopeCfg {
+// NewConfig returns a new datastore Config, with the boltdb stored in dataDir.
+// If dataDir is empty, a default value is used.
+func NewConfig(dataDir string) Config {
 	var dbpath string
 	if dataDir == "" {
```

Comment:
Hm... also see my comment on the previous commit; so if `dataDir` is the ONLY parameter that can be passed; do we need this function at all?

- Change `New()` to only accept a single argument (`path`)? 
- ☝️ Are we using the _dir_ anywhere, or only the full path to the _file_ ?
- `NewConfig` can be removed; it'd be part of the `New` constructor
- Maybe `Config` can even be un-exported if it's only used internally